src/front/js/store/flux.js

```diff
@@ -3,8 +3,32 @@ import jwt_decode from "jwt-decode";
 const getState = ({ getStore, getActions, setStore }) => {
 	return {
 		store: {
-			baseURL: "https://3001-aqua-yak-7fx7gv5u.ws-eu15.gitpod.io/api",
-			currentUser: {}
+			baseURL: "https://3001-copper-gazelle-52pduwl1.ws-eu16.gitpod.io/api",
+			currentUser: {},
+
+			product_id: {
+				category: "Computers",
+				shopname: "Amazon",
+				product_name: "product-name",
+				media: "https://cdn.pixabay.com/photo/2014/07/31/23/10/biker-407123__480.jpg",
+				price: 20,
+				text:
+					"Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet. "
+			},
+
+			message: null,
+			demo: [
+				{
+					title: "FIRST",
+					background: "white",
+					initial: "white"
+				},
+				{
+					title: "SECOND",
+					background: "white",
+					initial: "white"
+				}
+			]
```

Comment:
Esto podéis quitarlo, que era de ejemplo del boilerplate

Reply:
Done